lib/modules/manager/npm/extract/pnpm.ts

```diff
@@ -135,3 +137,48 @@ export async function detectPnpmWorkspaces(
     }
   }
 }
+
+export async function getPnpmShrinkwrap(filePath: string): Promise<LockFile> {
```

Comment:
```suggestion
export async function getPnpmLockfile(filePath: string): Promise<LockFile> {
```
we should rename, otherwise name is confusing

Reply:
We should drop the `shrinkwrap` usgae from everywhere since we only read from `pnpm-lock.yaml`.

Reply:
yes, but do in a separate refactor PR